plugins/http/test/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2021 SkyAPM org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '2.1'
+
+services:
+  mockoap:
+    image: arugaldocker/skywalking-mock-collector:1.0.0
```

Comment:
I think we should built the image on the fly (I prefer) or release this docker officially in Apache, otherwise other repo will copy and paste to use this image, which is bad

Reply:
Agree, this is what I used for testing. I forgot to change it.

Reply:
Hi @dmsolr do you have time to finish this？

Reply:
We need an official release for this. otherwise, arugaldocker/skywalking-mock-collector is a potential trademark issue.

Reply:
@arugal Do you want to initiate a release for this? Mostly, we need to resolve the license and notice files.

Reply:
> @arugal Do you want to initiate a release for this? Mostly, we need to resolve the license and notice files.

OK, I will finish this.

Reply:
via `ghcr.io/apache/skywalking-agent-test-tool/mock-collector:b01d4137fed862338d78a73b23435ccf425f9962`